packages/remix-oxygen/src/server.ts

```diff
@@ -33,6 +33,17 @@ export function createRequestHandler<Context = unknown>({
       });
     }
 
+    const url = new URL(request.url);
+
+    if (url.pathname.includes('//')) {
+      return new Response(null, {
+        status: 302,
```

Comment:
mmm .. the goal is to make sure we don't get into infinite redirect.

`404` would be the correct status code to return because we don't want to encourage bad url usage. However, it is not very user friendly to land on a not found page if it was just a typo.

`301` **permanently moved** would impact SEO but bot getting `//` from a public search is rare.

`302` **moved temporary**. Which is not true, this url is a malformed url. So this url shouldn't ever be active in future

Personally, I think `301` would be a better choice